### DISCUSSION TITLE
Feature request: make GitHub Co-Pilot free for educators.

### DISCUSSION THREAD

**Discussion by bloombar** [community/community#19285] [Copilot Conversations]
Dear @github-staff and @github-education,

GitHub Co-Pilot and other AI tools are changing how programmers work.  As a result, it is inevitably changing how students work and how we as educators teach those students and our expectations for how they complete their work.  Currently, **GitHub Co-Pilot is free for students, but not for educators**. This creates conflicts for educators, as we do not have access to the same assistive tools that our own students use to complete their work, we cannot explore innovative ways to use these tools in our teaching, and we cannot effectively advocate for changes to our institutions' codes of conduct, which often prohibits code copying.

This is an urgent problem.  Please seriously consider giving educators free license to use GitHub Co-Pilot as soon as possible.  A discussion about this is taking place on the Community discussion boards: https://github.com/github-community/Global-Campus-Teachers/discussions/92.

**Comment by D7EAD** [comment]
Hi!

I certainly agree that Github Copilot should be made available for free to educators and institutions! Have you considered contacting Github Education directly regarding this matter and if it may be an upcoming addition? If not, you can do so [here](https://support.github.com/contact/education).
